### PR TITLE
handle Hugo extensions (HGNC)

### PR DESCRIPTION
**Pull request by labkey-matthewb**

### CODE REVIEW

File: ontology/rdf.py [hunk 1]
```diff
@@ -138,44 +140,59 @@ def main():
...
-        if -1 == s.find("#"):
-            continue
-        lang = args.language
-        if isinstance(o, rdflib.term.BNode):
-            continue
-        elif isinstance(o, rdflib.term.Literal):
-            if o.language and o.language != args.language:
+    # initial pass:  just looking for owl#Class entries
+
+    for S, P, O in g:
```
Comment (Sigmonia) [inline ontology/rdf.py:148]: The names could be a little more descriptive

File: ontology/rdf.py [hunk 2]
```diff
@@ -138,44 +140,59 @@ def main():
...
+        c = subjects[s]
 
         if p.endswith("#prefLabel"):
             c.name = o
             c.synonyms[c.name.lower()] = True
-        elif p.endswith("#altLabel"):
+        elif p.endswith("/Hugo.owl#Approved_Symbol"):
+            if not c.name:
+                c.name = o
+                c.synonyms[c.name.lower()] = True
```
Comment (Sigmonia) [inline ontology/rdf.py:188]: should this be a method to ensure it matches up with the block above?

File: ontology/rdf.py [hunk 3]
```diff
@@ -138,44 +140,59 @@ def main():
...
+        o = str(O)
+        if s not in subjects:
+            continue
+        c = subjects[s]
 
         if p.endswith("#prefLabel"):
             c.name = o
             c.synonyms[c.name.lower()] = True
-        elif p.endswith("#altLabel"):
+        elif p.endswith("/Hugo.owl#Approved_Symbol"):
```
Comment (Sigmonia) [inline ontology/rdf.py:185]: combine with above as an `or`?

**Review by Sigmonia — APPROVED** [review]

**Review by labkey-tchad — APPROVED** [review]

File: ontology/rdf.py (hunk 1 above)
Comment (labkey-matthewb) [reply ontology/rdf.py:148]: I'll catch some of these suggestions in a follow on checkin.

**Pull request merged**